patches/openwrt/0900-replace-banner-with-freifunk.patch

```diff
@@ -0,0 +1,41 @@
+From: Nick Hainke <vincent@systemli.org>
+Date: Sat, 30 Sep 2017 10:35:19 +0200
+Subject: base-files: banner: replace openwrt banner with freifunk
+
+Replace openwrt banner with freifunk
+Signed-off-by: Nick Hainke <vincent@systemli.org>
+
+diff --git a/package/base-files/files/etc/banner b/package/base-files/files/etc/banner
+index 799f7e5..628447c 100644
+--- a/package/base-files/files/etc/banner
++++ b/package/base-files/files/etc/banner
+@@ -1,14 +1,17 @@
+-  _______                     ________        __
+- |       |.-----.-----.-----.|  |  |  |.----.|  |_
+- |   -   ||  _  |  -__|     ||  |  |  ||   _||   _|
+- |_______||   __|_____|__|__||________||__|  |____|
+-          |__| W I R E L E S S   F R E E D O M
++  _____        _  __             _
++ |  ___|      (_)/ _|           | |
++ | |_ _ __ ___ _| |_ _   _ _ __ | | __
++ |  _| '__/ _ \ |  _| | | | '_ \| |/ /
++ | | | | |  __/ | | | |_| | | | |   <
++ \_| |_|  \___|_|_|  \__,_|_| |_|_|\_\
```

Comment:
> Gluon is a modular framework for creating OpenWrt-based firmwares for wireless mesh nodes. Several Freifunk communities in Germany use Gluon as the foundation of their Freifunk firmwares.

While several Freifunk communities use Gluon to build their firmwares, it is not a dedicated Freifunk firmware framework.